scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -2073,14 +2073,16 @@ class FormatOps(
           style: ScalafmtConfig
       ): Option[OptionalBracesRegion] =
         ft.meta.leftOwner match {
-          case ParamClauseParent(t @ Defn.ExtensionGroup(_, _, b: Term.Block))
-              if isBlockStart(b, nft) =>
+          case ParamClauseParent(
+                t @ Defn.ExtensionGroup.internal.Impl(_, b: Term.Block)
+              ) if isBlockStart(b, nft) =>
             Some(new OptionalBracesRegion {
               def owner = Some(t)
               def splits = Some(getSplitsMaybeBlock(ft, nft, b))
               def rightBrace = blockLast(b)
             })
-          case t @ Term.If(_, thenp, _) if !nft.right.is[T.KwThen] && {
+          case t @ Term.If.internal.Impl(_, thenp, _, _)
```

Comment:
```suggestion
          case t : Term.If
```
and then just use t.thenp